app/templates/components/explore/side-bar.hbs

```diff
@@ -0,0 +1,69 @@
+<div class="ui fluid vertical menu">
+  <div class="map item">
+    {{g-map lat=37.7833 lng=-122.4167 zoom=2 class="explore"}}
+  </div>
+  <div class="item">
+    <div class="ui input">
+      <input type="text" placeholder="enter location">
+    </div>
+  </div>
+  <div class="item">
+    {{#ui-accordion}}
+      <span class="title">{{t 'Categories'}}</span>
+      <div class="content menu">
+        {{#each-in categories as |category subCategories| }}
+          <div class="link item {{if (eq filters.category category) 'active'}}">
+            <span role="button" {{action 'selectCategory' category}}>{{category}}</span>
```

Comment:
@niranjan94 This button triggers the action which triggers action `selectCategory` with it's parameter as `category` , changes the default value of category filter ( `NULL`) or some other category    to the current category?